src/main/fc/rc_modes.c

```diff
@@ -143,10 +148,22 @@ bool isRangeActive(uint8_t auxChannelIndex, const channelRange_t *range)
         return false;
     }
 
+    uint8_t channelIndex = auxChannelIndex + NON_AUX_CHANNEL_COUNT;
+
+    // On helicopter like aircraft, the throttle RC-channel is used 
+    // to indicate the headspeed to the FC. For this, AUX3 is mapped 
+    // to the throttle RC-channel. The headspeed is used to determine,
+    // which amount of collective pitch is needed to hover. woga65:
+#if defined(USE_VARIABLE_PITCH)
```

Comment:
I am not sure if this should be here. It should probably be a separate function called where it is needed.

The way it looks like now, you call a function that seems to imply being a simple read only check (called isSomehing) and now it has a random side effect on other parts of the code.